lesson6/README.md

```diff
@@ -0,0 +1,87 @@
+# Lesson 6 - How to Create an App
+
+##  Designing an app
+
+#### 1. Observe
+Open any app on your phone. It could be very simple or very complex; there is something to learn from both. While using it, ask questions. Some examples of questions could be:
+  1. What does the logo/name communicate to the users?
+  2. Are all the features easily discoverable?
+  3. In order to perform an action, how many times did you tap? Too many popups asking you to choose YES or NO?
+  4. How does the color scheme fit the overall feel?
+
+Read through the [Android Design Principles](https://developer.android.com/design/get-started/principles.html). It will help you think of more questions to ask.
+
+#### 2. Design
+Asking question while looking at an app can be incredibly helpful while trying to design something. Once you have an idea what your app is going to be about, make a first rought draft. Concentrate on the features you will be providing and the top-level navigation of the app. 
+
+Draw the screens on pieces of paper (or use one of the many online programs like fluidui). Test these mockups on your friends. It is recommended that you let your friends first explore the mockups without any instructions and observe what they see, think and do. Later, you can ask more specific questions about the features in your mockup.
+
+Next step is to integrate whatever feedback you received from the User Testing into your app design. At this stage, you can also decide on the color scheme of the app, logo, name etc. You could do cycles of user testing-->feedback integration--> testing again until you are satisfied with the app.
+
+Apps like FB Messenger or Snapchat would go through many periodic Usability Testing. It is important to make sure that your app remains sleek and intuitive while adding new features or editing existing features.
+
+#### 3. Testing after developing:
+Once you have a Minimum Viable Product (MVP) of your app, test again. But this time, it is a little different.
```

Comment:
*test it again.
Don't really need the "But this time, it is a little different."
Maybe mention trying to test it after adding little things